packages/react-native-web/src/exports/AccessibilityInfo/index.js

```diff
@@ -7,14 +7,21 @@
  * @flow
  */
 
-function emptyFunction() {}
-
 function isScreenReaderEnabled(): Promise<*> {
   return new Promise((resolve, reject) => {
     resolve(true);
   });
 }
 
+const prefersReducedMotionMedia = window.matchMedia('(prefers-reduced-motion: reduce)')
```

Comment:
This needs a canUseDOM guard for SSR context

Reply:
Okay, here you go 😊